api/src/main/java/org/openmrs/module/fhir2/providers/r4/MediaFhirResourceProvider.java

```diff
@@ -0,0 +1,34 @@
+/*
+ * This Source Code Form is subject to the terms of the Mozilla Public License,
+ * v. 2.0. If a copy of the MPL was not distributed with this file, You can
+ * obtain one at http://mozilla.org/MPL/2.0/. OpenMRS is also distributed under
+ * the terms of the Healthcare Disclaimer located at http://openmrs.org/license.
+ *
+ * Copyright (C) OpenMRS Inc. OpenMRS is a registered trademark and the OpenMRS
+ * graphic logo is a trademark of OpenMRS Inc.
+ */
+package org.openmrs.module.fhir2.providers.r4;
+
+import ca.uhn.fhir.rest.server.IResourceProvider;
+import lombok.AccessLevel;
+import lombok.Setter;
+import org.hl7.fhir.instance.model.api.IBaseResource;
+import org.hl7.fhir.r4.model.Media;
+import org.openmrs.module.fhir2.api.FhirMediaService;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.beans.factory.annotation.Qualifier;
+import org.springframework.stereotype.Component;
+
+@Component("mediaFhirR4ResourceProvider")
+@Qualifier("fhirResources")
+@Setter(AccessLevel.PACKAGE)
+public class MediaFhirResourceProvider implements IResourceProvider {
```

Comment:
We need some actual resource methods here, similar to the other `FhirResourceProvider` classes.